galsim/correlatednoise.py

```diff
@@ -1393,22 +1369,16 @@ def getCOSMOSNoise(file_name=None, rng=None, cosmos_scale=0.03, variance=0., x_i
     if file_name is None:
         file_name = os.path.join(meta_data.share_dir,'acs_I_unrot_sci_20_cf.fits')
     if not os.path.isfile(file_name):
-        raise IOError("The file '"+str(file_name)+"' does not exist.")
+        raise OSError("The file %r does not exist."%(file_name))
```

Comment:
I think I'd rather keep it separate.  The file not existing is a straightforward error that the user can probably deal with.  The next line should be an unusual occurrence that probably indicates some kind of corruption of the file.  I did add the text of the original error though, in case that's helpful for the user to figure out what was wrong.